packages/terra-tabs/tests/wdio/tabs-spec.js

```diff
@@ -82,4 +83,17 @@ Terra.describeViewports('Tabs - Large screen', ['large'], () => {
       Terra.validates.element();
     });
   });
+
+  describe('Additional Tabs after mount', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-tabs/tabs/tabs/additional-tabs');
+    });
+
+    Terra.it.matchesScreenshot('before');
+
+    it('Additional Tabs', () => {
+      $('#tabsWrapper-20').waitForExist(4000);
```

Comment:
This could be updated to 
```suggestion
      browser.waitForExist('#tabsWrapper-20', 4000);
```
Removed the need for the added global and matches our other tests.

Reply:
Done.